hazelcast/src/main/java/com/hazelcast/config/JoinConfig.java

```diff
@@ -202,28 +202,14 @@ public void verify() {
         if (getEurekaConfig().isEnabled()) {
             countEnabled++;
         }
+        Collection<DiscoveryStrategyConfig> discoveryStrategyConfigs = discoveryConfig.getDiscoveryStrategyConfigs();
+        if (discoveryStrategyConfigs.size() > 0) {
+            countEnabled++;
```

Comment:
Maybe `countEnabled += discoveryStrategyConfigs.size();`? I think we don't allow multiple discovery strategies at the same time.

Reply:
makes sense, updated. Thanks for the review Rafal, btw 👍